### DISCUSSION TITLE
AKT 2.0 Early Draft

### DISCUSSION THREAD

**Discussion by gosuri** [akash-network/community#32] [Economics]
As promised, the draft version of AKT 2.0 [proposal](https://github.com/gosuri/akt20/blob/main/README.md) is now available to be discussed by [sig-economics](https://github.com/akash-network/community/tree/main/sig-economics) in inaugural [meeting #1](https://github.com/akash-network/community/blob/main/sig-economics/meetings/1-2023-02-01.md) .

Please post your feedback here.

**Comment by sneakybusiness** [comment]
I really like this proposal. Very excited! But why burn AKT instead of giving that as well to AKT stakers? A burn benefits non stakers just as much. Would like to hear the reasoning behind this. Thanks!

**Reply by gosuri** [reply]
Stakers get a portion of incentives where non-stakes (holders) don't. By burning we are incentivizing non-stakers as well. 

Since burning removes AKT from the circulating supply, it reduces sell pressure as well w which in turn strengthens AKT security.

**Reply by sneakybusiness** [reply]
Thanks for the answer. Can't really complain just curious to know if my thought was considered. Burn + take income is of course an enormous upgrade. The other parts of the paper really important as well for the growth of the network. Great work!

**Reply by gosuri** [reply]
Thank you! The working group will upgrade this proposal to a paper with the details necessary for the implementation.

I'm also looking forward to an improved inflationary distribution mechanism.

**Reply by EtherDotBlue** [reply]
Quantstamp would love to audit the next version at partnership level pricing, and add real time threat detection agents from Hypernative. 💪🏻

**Comment by AntiquityStudio** [comment]
I'm still wrapping my head around it, but it's a very interesting read so far. 

In the stable payments section you are concerned about AKT being the dominant settlement token. In my ignorance, I just imagine a system where from the users perspective they pay in a stablecoin, but then the network uses osmosis or something to seamlessly switch it to AKT. 

What is the story/rationale behind having to whitelist tokens and having them as the token of payment/settlement, rather than having an interface whereby the user just has a wallet containing some cosmos-based stablecoin, and the contract is negotiated in terms reflecting the stablecoin value, but then internally the system just takes the tokens and converts them to the correct amount of AKT, using AKT as the actual token of payment/settlement? Any dollar or other stablecoin used would just be swapped on osmosis or elsewhere for AKT before the transaction runs.

**Reply by cloud-j-luna** [reply]
The idea is to increase the AKT value by a mechanism of burning. This way by decreasing the circulating amount, AKT gains value. This way AKT holders are benefited and the Network's security is not jeopardized by allowing "more convenient" stable payments.

**Reply by lechenghiskhan** [reply]
As far as the whitelist of currencies being supported, that's ultimately going to be managed via governance and community support. On the back end, you're right in that the mechanism will probably leverage Osmosis to swap. 

One of the key reasons for highlighting USDC (a stable currency) first and foremost, is because having a familiar, stable currency to manage payments (tenants) and settlements (providers) is a huge quality of life gain. Understanding how much deployments cost in dollar terms or how much you can make per month as a provider in dollar terms would solve one of the biggest challenges of a volatile asset as the only medium of payment.

**Reply by gosuri** [reply]
Thank you for the feedback.

To summarize your question, Why do we need to replace the payment token from AKT with a whitelisted one (Stable) instead of leaving the payment token as AKT and wrapping the fee with a whitelisted currency that swaps (from a DEX) to AKT in real-time?

This could be achieved by pricing the workloads in the stablecoin and pegging to AKT, where the amount of AKT transferred from Tenant to Provider could differ (based on AKT volatility).

Your approach is certainly valid; we initially thought about the **_peg model_**. The only downside is reliance on a DEX for every deployment and complications in slippage calculations.

We will discuss this approach in detail and weigh the pros and cons in the SIG meeting tomorrow.

**Reply by AntiquityStudio** [reply]
Yes, that's exactly my question, thank you. Looking forward to hearing more tomorrow.

**Comment by hewi333** [comment]
I love the chart in the Incentive Distribution Pool section. With current network spend, inflationary rewards will be the major contributor to the Incentive Distribution Pool until Tenant & Provider fees start rolling in. What kind of tax to inflationary rewards have been considered to make this successful? Also, how do you see the Developer Fund vs. the existing Community Pool that was increased to 10% from 2%? Thanks and awesome work!

**Reply by gosuri** [reply]
The `Incentive Distribution Pool` distribution values will be determined by the Working Group after carefully modeling the impact. Some considerations:

1) Cost of Security: Explore ways to optimize security cost by modeling various attack vectors and explore using mesh security and/or shared security when possible.
2) Sell pressure providers: Are are sufficient incentives for providers to hold AKT for long?
3) Length of incentives and critical milestones.

**Reply by lechenghiskhan** [reply]
The current % going to the Community Pool would probably be rebalanced. 

The current Community Pool is a bit of a catch-all that can be tapped for anything from liquidity incentives to funding developers building/growing Akash Network. In the proposal, the participants would be more purpose-specific (developer fund, for instance) vs a generic community pool functioning as a catch-all.

**Reply by sneakybusiness** [reply]
Do you see any risk with shared security for the value of AKT? Or would take income only go to AKT stakers in that case and part of inflationary rewards to lets say ATOM validators? Sorry for only asking about AKT value accrual but as an AKT investor its an important subject for me. Thanks!

**Comment by bmenzalji** [comment]
The following two points are for consideration and discussion with community in regards to Akash's Tokenomics 2.0 Update.

**Summarize Suggestions:**

1. Split a % of Take Fee to reward for pre-existing and established tools that drive demand via performance-based user activation
2. Introduce tier-based Make Fee for providers to reward activation of their offerings (CPU, IP, Perm Storage, Audited Attributes, etc), while increasing provider reputation for those that have fulfilled complete onboarding tasks.

**Performance rewards for Clients that drive demand:**

The proposal for multi-currency settlement in it's current state to fund the incentivized distribution pool through Tenant, Provider and Network is missing a critical item.  We need to position Akash to attract demand-generating onboarding mechanisms. The core objective of the following is for Akash to optimize for demand first through performance-based rewards.  This widens onboarding funnel for network growth, while growing more value to AKT 

**Philosophy: Public Cloud Utility = Open Cloud Economy**
As a permissionless network, the strength of Akash's product positioning is to ultimately become an aggregator/hub for all compute services, both centralized/decentralized. To reach cloud parity, we must invite everyone to plug into Akash Network, and should reward for performance when it comes to user activation.  This attracts and opens the network up for pre-existing products/solutions to integrate Akash directly into their tech stack in order to extend Akash Cloud marketplace to their users, not just fund plans that stem from working groups to accelerate demand.  While also beneficial, Akash Network's cloud economy greatly benefits in a variety of use cases.  While too many to list, we can add one example for color:

- Dev Funds a web3 service to plug into Akash Network's marketplace
- Established web2 client with 10s of thousands of users plugs that service in directly into their offering via the marketplace to expand their web3 offering tool suite

**Context**:
We must first recognize that community tools, such as Terraform Provider, Cloudmos, Spheron Network, Fleek, etc have lowered the barrier to entry for tenants onto Akash.  Their impact is felt or will be soon.  We must rewards for demand-side performance specifically, and the current mechanisms outlined do not do that.

For example:
There is an deployment console of a major organization (let's refer to this as Serafin Corp) with a 10s-of-thousands of users.  They are interested in expanding their offering to tap into Akash Network's marketplace.  

How does Akash position itself to entice decision makers to greenlight this initiative?  Compute is a commodity, and should be priced as such, but isn't with CSP's.  This is a value-add to their customer base to drive down compute pricing (CPU and GPU) and saves their customers money, while positioning Serafin to upsell them on managed services that are higher in margin.  It's a winning strategy because this decision making process and buying behavior exists in the marketplace already; give away compute for higher-margin SaaS products.

However, to reward Serafin, a portion of Take fees (10%?) would go directly to Serafin to fund overhead/support for this integration and user activation.  

**Why it's beneficial:**

- It pays for itself: This solves for Serafin's resources to fund this initiative (technical, support, marketing, etc), and rewards them for performance; the more of their users utilize their services, the more they earn from users accessing Akash Network.  
- Serafin's integration for product positioning allows them to retain their customers looking to save on cloud and is a strong value proposition with a looming recession.  
- Akash Network benefits from accessing mid-market and enterprise solutions to activate a pre-existing customer base they own.  It also incentives Serafin to invest in tooling/services that Akash Network is missing to support this initiative, well-beyond just access Akash Network's multi-cloud, which accelerates onboarding ramps that optimize for demand.

Bonus: The feedback loop occurs because it also incentives organizations of all sizes to contribute to Akash Networks needs that they will need; ie private repository support, automation, observability, etc.  This compounds development by accelerating timelines that benefit everyone in order for Akash Network to fullfill it's masterplan:
![image](https://user-images.githubusercontent.com/25206376/215987450-a86fd795-9c03-4fd4-9595-06652e3abbbb.png)

Inspiration Source:
- Contract Service Revenue from Canto: https://mirror.xyz/0x43617d5d3ab33bCC37F0fE29d969b0351514f377/Zy1QX5C_i3CuLQVuI1PO7L2IL0hc_Fg-4uQiVg1NoKY

**Provider Subsidies**

Provider subsidies are sustainable only if demand is solved for, which must include Akash Network having a reward mechanism to invite pre-existing deployment tools that developers are familiar with and love.  This would act as a catalyst to invite established tools to integrate Akash Network immediatel and align with our values of being public utility cloud.  That being said, some of the challenges Praetor and we have faced is onboarding cloud providers that do not enable full functionality of their capabilities; ie lack of IP's and persistent storage on the marketplace is evident.

To solve for this, the provider Make Fee should be divided into five **initial** components that benefit the integrity and quality of providers on the network itself (these components can increase/decrease over time based on Akash Network Provider capabilities):

For example: Bozanich Cloud is an established data center.  Akash Network would benefit from having them offer:
- Compute 
- Static IP's
- Bandwidth
- Permanent Storage
- Auditing their attributes

Assuming the Make Fee is in equal parts in this example, 20% of the make fee would be accrue in their account assuming those services are enabled, which incentivizes them to onboard successfully to unlock full rewards.  If providers do not audit their attributes, it hinders the quality of the network to establish reputational trust, and thus 80% of Make Fee is realized.  The remaining 20% is up for debate, but a suggestion would be to allow it to accrue in escrow for a pre-defined time (1-3 months?) and if that attribute is not realized, the funds are forfeited and funnel to the incentivized distribution pool normally.

**Comment by ghost** [comment]
Agreed with AKT token initial shortcomings, 

I Support Take and Make Fees, I also support of Stable Payment and Settlement with Multi-Currency Support as it can give a more viable solution for users to use akash

I believe these will make AKT token accrue value overtime.

I got a few questions/thoughts

What would happen if Akash do multicurrency support and those multi tokens are used to purchase AKT?** will that impact demand or else? 

If AKT stakers get fees collected, how would stakers know they are getting more than the normal APR? is Keplr or Mintscan going to show “APY on your Staked assets is XXX%” or should  the community do a bigger effort in educating newcommers on the benefits of holding AKT?

**Comment by nimnathw** [comment]
How can I retrieve price, quality (up-time) and quantity data for providers and users? Shouldn't we have a rough idea of the demand curve and the supply curve before we can estimate the effect of new tokenomics?
